src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -55,13 +47,44 @@ void createStation() {
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역 목록을 조회한다.")
+    @Test
+    void findStationNames() {
+        //given
+        List<Map<String, String>> stations = new ArrayList<>();
+        stations.add(Map.of("name", "부평구청역"));
+        stations.add(Map.of("name", "가산디지털단지역"));
+
+        stations.forEach(StationCommonApi::createStation);
```

Comment:
```suggestion
        createStation("부평구청역");
        createStation("가산디지털단지역");
```

각각의 역을 만드는 것을 표현해준다면, 인수자가 더 빠르게 파악하지 않을까하네요~